src/internal/clusterstate/v2.8.0/pfsdb.go

```diff
@@ -109,7 +105,7 @@ func createReposTable(ctx context.Context, tx *pachsql.Tx) error {
 	}
 	// Create a trigger that notifies on changes to pfs.repos
 	// This is used by the PPS API to watch for changes to repos
-	if _, err := tx.ExecContext(ctx, generateTriggerFunctionStatement("pfs", "repos", ReposChannelName)); err != nil {
+	if _, err := tx.ExecContext(ctx, generateTriggerFunctionStatement("pfs", "repos", pfsdb.ReposChannelName)); err != nil {
```

Comment:
This shouldn't import from pfsdb. Should be immutable, so just use a string literal.

Reply:
You're right (because it can change over time.) We should just redefine it here for the scope of this package too.